goat/utils/database_helper.py

```diff
@@ -0,0 +1,100 @@
+import os
+import select
+from dataclasses import dataclass
+from typing import Callable
+
+import pandas as pd
+import psycopg2
+from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
+from sqlalchemy import MetaData, Table, create_engine, insert, text
+from sqlalchemy.orm import sessionmaker
+
+POSTGRES_USER = os.environ.get("POSTGRES_USER")
+POSTGRES_PASSWORD = os.environ.get("POSTGRES_PASSWORD")
+POSTGRES_DB = os.environ.get("POSTGRES_DB")
+POSTGRES_IP = os.environ.get("POSTGRES_IP")
+POSTGRES_PORT = os.environ.get("POSTGRES_PORT")
```

Comment:
Probably, this should be part of `DatabaseHelper` class with additional check that these variables set